Documentation/Index.rst

```diff
@@ -25,45 +26,57 @@ Quick links
 - :ref:`Writing reST <h2document:Formatting-with-reST>`
 
 
-
-Latest documentation news
--------------------------
+Tips - did you know?
+------------------------------
 
 .. sidebar:: New to TYPO3?
 
-   If you are new to TYPO3, please start by looking at the
+   If you are new to TYPO3, please start by looking at the :ref:`t3start:start`
+   and the other
    :ref:`Tutorials <tutorials>`, which will help you progressively
    increase your knowledge of TYPO3.
 
    When you feel ready, you should quickly get an overview of what is
    within :ref:`TYPO3 explained <t3coreapi:start>` and see how this document
-   relates to the other :ref:`References <references>`.
+   relates to other :ref:`references`.
 
 .. End of sidebar
 
--  2019-03-08 :ref:`news-2019-03-08-2`
--  2019-03-08 :ref:`news-2019-03-08`
--  2019-02-14 :ref:`news-2019-02-14`
--  2018-11-13 :ref:`news-2018-11-13`
--  2018-11-05 :ref:`news-2018-11-05`
+-  Most manuals are available for several TYPO3 versions. :ref:`Read more ... <usage-version-selector>`
+-  You can get support for TYPO3 via StackOverflow or Slack. `Read more ... <https://typo3.org/help>`__
+-  You can click on the "Edit me on GitHub" button to edit a page. :ref:`Read more ... <h2document:docs-contribute>`
 
--  :ref:`All the news <news>`
+.. rst-class:: horizbuttons-primary-m
+
+-  :ref:`More TYPO3 tips <Tip-of-the-day>`
+-  :ref:`More usage tips <Tip-of-the-day>`
```

Comment:
Whats the difference to line 51?

Reply:
That was an error. I removed the extra link. Pushed new commit.